flutter_local_notifications/android/src/main/java/com/dexterous/flutterlocalnotifications/FlutterLocalNotificationsPlugin.java

```diff
@@ -245,15 +243,20 @@ private static void scheduleNotification(Context context, final NotificationDeta
         } else {
             AlarmManagerCompat.setExact(alarmManager, AlarmManager.RTC_WAKEUP, notificationDetails.millisecondsSinceEpoch, pendingIntent);
         }
+    }
+
+    private static void scheduleNotifications(Context context, final List<NotificationDetails> notificationDetails, Boolean updateScheduledNotificationsCache) {
```

Comment:
I like your overload refactor, but this could actually just be `scheduleNotificationsWithCacheUpdate` and then not have a boolean argument. But this would require more code changes :)